### DISCUSSION TITLE
Chromium build: No support for pq-algorithms

### DISCUSSION THREAD

**Discussion by ellawk** [open-quantum-safe/oqs-demos#219] [General]
Hi! I just built the browser (Chromium for Linux) with oqs-boringssl by following the instructions on this page. It looks right, and I don't get any errors during the process. But, the browser does only support the "old" algorithms, none of the pq-algorithms. It says in the instruction (after step 5) "If any error occurs in this step, Chromium will build fine, just without support for quantum-safe crypto, i.e., only the final testing steps below will fail." So I guess this the case. My question is, do you have any idea where I should start looking for errors to fix?

**Comment by baentsch** [comment]
Tagging @Raytonne . 

@ellawk If you'd simply be looking for a QSC-enabled browser (?) please consider using epiphany: There's even a [pre-build docker image](https://hub.docker.com/repository/docker/openquantumsafe/epiphany) ready-to-run (and test against the interop test server at https://test.openquantumsafe.org).

**Comment by Raytonne** [comment]
@ellawk Check the Windows build instructions [Step 5. Enable Quantum-Safe Crypto](https://github.com/open-quantum-safe/oqs-demos/blob/main/chromium/README-Windows.md#5-enable-quantum-safe-crypto)), it's up to date. Linux and Windows instructions are essentially the same
We'll update linux build instructions when we have time.

**Reply by Raytonne** [reply]
Several notes for Linux: 
1. when you edit `/third_party/boringssl/BUILD.gn`, libs should be
`libs = ["//third_party/boringssl/src/oqs/lib/liboqs.a"]`
2. use ninja instead of msbuild to build liboqs on Linux. script to build liboqs on Linux: https://github.com/open-quantum-safe/oqs-demos/blob/main/chromium/scripts/build_liboqs.sh

**Comment by Raytonne** [comment]
Linux instructions have been updated.
